translations/player_turns.tex/es.po

```diff
@@ -189,6 +258,10 @@ msgid ""
 "}\n"
 "\n"
 msgstr ""
+"\\note{5}{La Facción Necrópolis \\includesvg[height=10px]{\\svgs/necro-note.svg} ignora los efectos de Moral.\n"
```

Comment:
```suggestion
"\\note{4}{La Facción Necrópolis \\includesvg[height=10px]{\\svgs/necro-note.svg} ignora los efectos de Moral.\n"
```
You can make the box smaller. You could also try with `3` and see what works better.